docs/local-development.md

```diff
@@ -117,7 +117,7 @@ We wish to keep backwards-compatibility as often as possible, as well as make
 the code configurable, so most new functionality should be controllable via
 configuration options.
 
-If you wish to add one, add it to `lib/config/definitions.js` and then add documentation to `website/docs/_posts/2017-10-05-configuration-options.md`.
```

Comment:
why that?

Reply:
This looks like an old link, change looks good